src/pyeconet/api.py

```diff
@@ -196,7 +196,10 @@ async def _get_location(self) -> List[Dict]:
             ) as resp:
                 if resp.status == 200:
                     _json = await resp.json()
-                    _LOGGER.debug(_json)
+                    if _LOGGER.getEffectiveLevel() == logging.DEBUG:
+                        _LOGGER.debug("Logging json to locations_debug.json")
+                        with open('locations_debug.json', 'w') as debug_file:
```

Comment:
When this library is used in Home Assistant where will this file be written? Seems like it will result in making it more difficult to get logs from users.